### PR TITLE
[New Rule] Potential SharpRdp Detected

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/526

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
**Alert Example:**

image

File: rules/windows/lateral_movement_rdp_sharprdp_target.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2020/11/11"
+ecs_version = ["1.6.0"]
+maturity = "development"
```
Comment (bm11100) [inline rules/windows/lateral_movement_rdp_sharprdp_target.toml:4]:
```suggestion
maturity = "production"
```

File: rules/windows/lateral_movement_rdp_sharprdp_target.toml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+query_schema_validation = false
+updated_date = "2020/11/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies behavior execution of SharRdp a tool that can be used to perform authenticated command execution against a
+remote target via Remote Desktop Protocol for the purposes of lateral movement.
```
Comment (bm11100) [inline rules/windows/lateral_movement_rdp_sharprdp_target.toml:12]:
```suggestion
Identifies behavior of SharpRDP, which is a tool that can be used to perform authenticated command execution against a
remote target via Remote Desktop Protocol (RDP) for the purposes of lateral movement.
```

File: rules/windows/lateral_movement_rdp_sharprdp_target.toml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential SharpRdp Detected"
```
Comment (bm11100) [inline rules/windows/lateral_movement_rdp_sharprdp_target.toml:18]:
```suggestion
name = "Potential SharpRDP Behavior"
```

File: rules/windows/lateral_movement_rdp_sharprdp_target.toml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    "https://github.com/sbousseaden/EVTX-ATTACK-SAMPLES/blob/master/Lateral%20Movement/LM_sysmon_3_12_13_1_SharpRDP.evtx",
+]
+risk_score = 73
+rule_id = "8c81e506-6e82-4884-9b9a-75d3d252f967"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+/* Incoming RDP followed by RunMRU new string value set to cmd, powershell, taskmgr or tsclient followed by process execution within 1m */
```
Comment (bm11100) [inline rules/windows/lateral_movement_rdp_sharprdp_target.toml:30]:
```suggestion
/* Incoming RDP followed by a new RunMRU string value set to cmd, powershell, taskmgr or tsclient, followed by process execution within 1m */
```

File: rules/windows/lateral_movement_rdp_sharprdp_target.toml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2020/11/11"
+ecs_version = ["1.6.0"]
+maturity = "production"
+query_schema_validation = false
```
Comment (bm11100) [inline rules/windows/lateral_movement_rdp_sharprdp_target.toml:5]:
```suggestion

```

Comment (bm11100) [reply rules/windows/lateral_movement_rdp_sharprdp_target.toml:5]: since its no longer in dev we'll want to remove this, otherwise looks good.

**Review by bm11100 — APPROVED** [review]
just the one comment

**Review by devonakerr — APPROVED** [review]

**Pull request merged**